posthog/src/main/java/com/posthog/internal/replay/RRStyle.kt

```diff
@@ -19,4 +19,6 @@ public data class RRStyle(
     public var paddingLeft: Int? = null,
     public var paddingRight: Int? = null,
     public var bar: String? = null,
+    public var iconLeft: String? = null,
+    public var iconRight: String? = null,
```

Comment:
they hold a base64 value similar to `backgroundImage`, if present in text fields, they hold a small icon at the start (before the text), or at the end.

Reply:
@pauldambra we don't have that, shall we add it?

> Rate and review has no icon

Reply:
🤔 seems good to add to me... 👍

Reply:
Ah, I'm catching up here maybe... is it a custom library 🤔 

I don't know how much we want to bake custom library stuff into the schema. 

Willing to be led by you - I've no idea how popular this library is

Reply:
Shall we split things and get the padding fix in and then follow up with this icon change if we decide to do it?

Reply:
Its not a custom library, the icon is custom, but the drawables on the 4 possible positions are part of the UI system as well.
https://developer.android.com/reference/android/widget/TextView#getCompoundDrawables()

Reply:
cool beans, I'll add this to schema/transformer 👍 

will need to figure out what the defaults/rules are around display but that's just mechanical